families/supplychain/postgreSQL/tables_sample_data.sql

```diff
@@ -0,0 +1,100 @@
+-- Definition of tables for Supply Chain DB
+-- Tables to create
+
+
+CREATE TABLE block (
+	block_id			char(128) primary key,		-- primary key
+	block_num			integer, 
+	state_root_hash		char(64)
+);
+
+
+CREATE TABLE record (
+	id 					bigserial primary key, 		-- primary key
+	start_block_num		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	end_block_num		integer, 		-- This field is populated in response to events from the State Delta Subscription service, but is loosely coupled to the block_history table.
+	identifier			varchar(128), 	-- The natural key of the record, serial number of attached sensor identifier.
+	creation_time		bigint,			-- The time the record was created
```

Comment:
The only definition I could find in the REST API specs was "int". I will seek out this info and place in here as soon as I can.